src/Microsoft.Data.SqlClient/netcore/src/Resources/StringsHelper.cs

```diff
@@ -134,1041 +130,31 @@ internal partial class StringsHelper
-            internal const string DataColumnMapping_SourceColumn = @"Source column name - case sensitive.";
-            internal const string DataColumnMappings_Count = @"The number of items in the collection";
-            internal const string DataColumnMappings_Item = @"The specified DataColumnMapping object.";
-            internal const string DataTableMapping_ColumnMappings = @"Individual columns mappings when this table mapping is matched.";
-            internal const string DataTableMapping_DataSetTable = @"DataTable.TableName";
-            internal const string DataTableMapping_SourceTable = @"The DataTableMapping source table name. This name is case sensitive.";
-            internal const string DataTableMappings_Count = @"The number of items in the collection";
-            internal const string DataTableMappings_Item = @"The specified DataTableMapping object";
             internal const string DbDataAdapter_DeleteCommand = @"Used during Update for deleted rows in DataSet.";
             internal const string DbDataAdapter_InsertCommand = @"Used during Update for new rows in DataSet.";
             internal const string DbDataAdapter_SelectCommand = @"Used during Fill/FillSchema.";
             internal const string DbDataAdapter_UpdateCommand = @"Used during Update for modified rows in DataSet.";
             internal const string DbDataAdapter_RowUpdated = @"Event triggered before every DataRow during Update.";
             internal const string DbDataAdapter_RowUpdating = @"Event triggered after every DataRow during Update.";
-            internal const string DbDataAdapter_UpdateBatchSize = @"Number of rows to batch together before executing against the data source.";
-            internal const string DbTable_Connection = @"Connection used if the the Select/Insert/Update/DeleteCommands do not already have a connection.";
-            internal const string DbTable_DeleteCommand = @"Used during Update for deleted rows in the DataTable.";
-            internal const string DbTable_InsertCommand = @"Used during Update for new rows in the DataTable.";
-            internal const string DbTable_SelectCommand = @"Used during Fill.";
-            internal const string DbTable_UpdateCommand = @"Used during Update for modified rows in the DataTable.";
-            internal const string DbTable_ReturnProviderSpecificTypes = @"Should Fill return provider specific values or common CLSCompliant values.";
-            internal const string DbTable_TableMapping = @"How to map source table to DataTable.";
-            internal const string DbTable_ConflictDetection = @"How are the Insert/Update/DeleteCommands generated when not set by the user.";
-            internal const string DbTable_UpdateBatchSize = @"Number of rows to batch together before executing against the data source.";
-            internal const string DbConnectionString_ConnectionString = @"The connection string used to connect to the Data Source.";
```

Comment:
Could you include the `DbConnectionString_` const strings related to SqlClient?

Reply:
These are unused strings.
SqlConnection_ConnectionString is used for instance.